storm-client/src/jvm/org/apache/storm/streams/PairStream.java

```diff
@@ -380,6 +382,26 @@
         return partitionBy(KEY).updateStateByKeyPartition(stateUpdater);
     }
 
+    /**
+     * Groups the values of this stream with the values having the same key from the other stream.
+     * If stream1 has values - (k1, v1), (k2, v2), (k2, v3)
+     * and stream2 has values - (k1, x1), (k1, x2), (k3, x3)     *
```

Comment:
please format so that its easy to read in javadocs and remove the extra `*` at the end of this line.